### PR TITLE
Fix systemtags overflow

### LINKED ISSUE #19859: Tags in detailview looks ugly
### Steps to reproduce
add some tags to a file

### Expected behaviour
Tag should get enough room in the detailview.

### Actual behaviour
Tags lay over the other thing of the detail pane

## Server configuration detail

**Operating system:** Linux 5.5.6-1-default #1 SMP Mon Feb 24 09:02:31 UTC 2020 (4a830b1) x86_64

**Webserver:** Apache (fpm-fcgi)

**Database:** mysql 10.4.12

**PHP version:** 

7.4.3
Modules loaded: Core, date, libxml, pcre, filter, hash, Reflection, SPL, session, SimpleXML, standard, xml, mysqlnd, cgi-fcgi, apcu, bcmath, bz2, calendar, ctype, curl, dba, dom, enchant, mbstring, fileinfo, ftp, gd, gettext, gmagick, gmp, ice, iconv, intl, json, lzf, exif, memcached, mysqli, openssl, pcntl, PDO, pdo_mysql, pdo_sqlite, zlib, posix, readline, redis, smbclient, snmp, sockets, sqlite3, tidy, tokenizer, uuid, xmlreader, xmlrpc, xmlwriter, xsl, zip, Phar, libsmbclient, Zend OPcache

**Nextcloud version:** 18.0.1 - 18.0.1.3

**Updated from an older Nextcloud/ownCloud or fresh install:** 

**Where did you install Nextcloud from:** unknown

<details><summary>Signing status</summary>

Array
(
)

</details>

<details><summary>List of activated apps</summary>

```
Enabled:
 - activity: 2.11.0
 - admin_audit: 1.8.0
 - bookmarks: 2.3.4
 - bruteforcesettings: 1.5.0
 - calendar: 2.0.2
 - checksum: 0.4.4
 - circles: 0.18.3
 - cloud_federation_api: 1.1.0
 - comments: 1.8.0
 - contacts: 3.2.0
 - dav: 1.14.0
 - event_update_notification: 1.0.1
 - federatedfilesharing: 1.8.0
 - federation: 1.8.0
 - files: 1.13.1
 - files_accesscontrol: 1.8.1
 - files_automatedtagging: 1.8.0
 - files_external: 1.9.0
 - files_fulltextsearch: 1.4.1
 - files_fulltextsearch_tesseract: 1.4.0
 - files_markdown: 2.2.0
 - files_pdfviewer: 1.7.0
 - files_rightclick: 0.15.2
 - files_sharing: 1.10.1
 - files_trackdownloads: 1.7.0
 - files_trashbin: 1.8.0
 - files_versions: 1.11.0
 - files_videoplayer: 1.7.0
 - firstrunwizard: 2.7.0
 - flowupload: 0.1.8
 - fulltextsearch: 1.4.1
 - fulltextsearch_elasticsearch: 1.5.0
 - groupfolders: 6.0.1
 - issuetemplate: 0.6.0
 - logreader: 2.3.0
 - lookup_server_connector: 1.6.0
 - mail: 1.1.3
 - maps: 0.1.5
 - metadata: 0.11.1
 - music: 0.13.0
 - news: 14.1.3
 - nextbackup: 19.12.1
 - nextcloud_announcements: 1.7.0
 - notes: 3.1.5
 - notifications: 2.6.0
 - oauth2: 1.6.0
 - passman: 2.3.5
 - password_policy: 1.8.0
 - phonetrack: 0.6.2
 - photos: 1.0.0
 - polls: 1.3.0
 - previewgenerator: 2.2.0
 - pride: 0.0.3
 - privacy: 1.2.0
 - provisioning_api: 1.8.0
 - recommendations: 0.6.0
 - serverinfo: 1.8.0
 - settings: 1.0.0
 - sharebymail: 1.8.0
 - sharingpath: 0.1.1
 - support: 1.1.0
 - survey_client: 1.6.0
 - suspicious_login: 3.1.0
 - systemtags: 1.8.0
 - talk_simple_poll: 1.1.0
 - tasks: 0.12.1
 - text: 2.0.0
 - twofactor_backupcodes: 1.7.0
 - updatenotification: 1.8.0
 - user_ldap: 1.8.0
 - user_usage_report: 1.2.0
 - viewer: 1.2.0
 - workflow_script: 1.3.1
 - workflowengine: 2.0.0
Disabled:
 - accessibility
 - analytics
 - apporder
 - encryption
 - extract
 - files_linkeditor
 - files_retention
 - impersonate
 - spreed
 - theming
 - w2g2
 - workflow_pdf_converter

```
</details>

<details><summary>Configuration (config/config.php)</summary>

```
{
    "instanceid": "***REMOVED SENSITIVE VALUE***",
    "passwordsalt": "***REMOVED SENSITIVE VALUE***",
    "secret": "***REMOVED SENSITIVE VALUE***",
    "trusted_domains": [
        "nextcloud.mybase.selfhost.de",
        "cloudnext.mybase.selfhost.de"
    ],
    "datadirectory": "***REMOVED SENSITIVE VALUE***",
    "tempdirectory": "\/var\/tmp\/nextcloudtemp",
    "overwrite.cli.url": "https:\/\/nextcloud.mybase.selfhost.de",
    "knowledgebaseenabled": true,
    "dbtype": "mysql",
    "version": "18.0.1.3",
    "dbname": "***REMOVED SENSITIVE VALUE***",
    "dbhost": "***REMOVED SENSITIVE VALUE***",
    "dbport": "",
    "dbtableprefix": "oc_",
    "dbuser": "***REMOVED SENSITIVE VALUE***",
    "dbpassword": "***REMOVED SENSITIVE VALUE***",
    "installed": true,
    "updater.release.channel": "beta",
    "appstore.experimental.enabled": true,
    "maintenance": false,
    "loglevel": 3,
    "debug": false,
    "htaccess.RewriteBase": "\/",
    "enable_previews": true,
    "preview_max_x": 1080,
    "preview_max_y": 1920,
    "preview_max_filesize_image": 100,
    "preview_libreoffice_path": "\/usr\/bin\/libreoffice",
    "preview_office_cl_parameters": "--headless --nologo --nofirststartwizard --invisible --norestore --convert-to pdf --outdir ",
    "enabledPreviewProviders": [
        "OC\\Preview\\PNG",
        "OC\\Preview\\JPEG",
        "OC\\Preview\\GIF",
        "OC\\Preview\\BMP",
        "OC\\Preview\\XBitmap",
        "OC\\Preview\\MP3",
        "OC\\Preview\\TXT",
        "OC\\Preview\\MarkDown",
        "OC\\Preview\\TIFF",
        "OC\\Preview\\SVG",
        "OC\\Preview\\Postscript",
        "OC\\Preview\\OpenDocument",
        "OC\\Preview\\PDF",
        "OC\\Preview\\Font",
        "OC\\Preview\\Epub",
        "OC\\Preview\\StarOffice",
        "OC\\Preview\\TIFF"
    ],
    "mail_from_address": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpmode": "smtp",
    "mail_smtpauthtype": "LOGIN",
    "mail_domain": "***REMOVED SENSITIVE VALUE***",
    "log_type": "syslog",
    "logfile": "\/var\/log\/nextcloud.log",
    "logfilemode": 416,
    "log_rotate_size": 104857600,
    "syslog_tag": "Nextcloud",
    "memcache.distributed": "\\OC\\Memcache\\Redis",
    "memcache.local": "\\OC\\Memcache\\Redis",
    "filelocking.enabled": "true",
    "filelocking.ttl": 3600,
    "memcache.locking": "\\OC\\Memcache\\Redis",
    "redis": {
        "host": "***REMOVED SENSITIVE VALUE***",
        "port": 6379,
        "timeout": 0,
        "dbindex": 0,
        "password": "***REMOVED SENSITIVE VALUE***"
    },
    "memcached_servers": [
        [
            "127.0.0.1",
            11211
        ]
    ],
    "data-fingerprint": "790e9a77b6ee76b2af214c8f58d75dfd",
    "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpauth": 1,
    "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
    "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpsecure": "tls",
    "mysql.utf8mb4": true,
    "allow_user_to_change_display_name": true,
    "auth.bruteforce.protection.enabled": true,
    "ldapIgnoreNamingRules": false,
    "ldapProviderFactory": "OCA\\User_LDAP\\LDAPProviderFactory",
    "theme": ""
}
```
</details>

**Are you using external storage, if yes which one:** local/smb/sftp/...

**Are you using encryption:** 

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

<details><summary>LDAP configuration (delete this par if not used)</summary>

```
background_sync_interval: 1800enabled: yesinstalled_version: 1.8.0s01_lastChange: 1579345588s01has_memberof_filter_support: 0s01home_folder_naming_rule: s01last_jpegPhoto_lookup: 0s01ldap_agent_password: s01ldap_attributes_for_group_search: s01ldap_attributes_for_user_search: s01ldap_backup_host: s01ldap_backup_port: s01ldap_base: s01ldap_base_groups: s01ldap_base_users: s01ldap_cache_ttl: 600s01ldap_configuration_active: 0s01ldap_default_ppolicy_dn: s01ldap_display_name: displayNames01ldap_dn: s01ldap_dynamic_group_member_url: s01ldap_email_attr: s01ldap_experienced_admin: 0s01ldap_expert_username_attr: s01ldap_expert_uuid_group_attr: s01ldap_expert_uuid_user_attr: s01ldap_ext_storage_home_attribute: s01ldap_gid_number: gidNumbers01ldap_group_display_name: cns01ldap_group_filter: s01ldap_group_filter_mode: 0s01ldap_group_member_assoc_attribute: s01ldap_groupfilter_groups: s01ldap_groupfilter_objectclass: s01ldap_host: s01ldap_login_filter: s01ldap_login_filter_mode: 0s01ldap_loginfilter_attributes: s01ldap_loginfilter_email: 0s01ldap_loginfilter_username: 1s01ldap_nested_groups: 0s01ldap_override_main_server: s01ldap_paging_size: 500s01ldap_port: s01ldap_quota_attr: s01ldap_quota_def: s01ldap_tls: 0s01ldap_turn_off_cert_check: 0s01ldap_turn_on_pwd_change: 0s01ldap_user_avatar_rule: defaults01ldap_user_display_name_2: s01ldap_user_filter_mode: 0s01ldap_userfilter_groups: s01ldap_userfilter_objectclass: s01ldap_userlist_filter: s01use_memberof_to_detect_membership: 1types: authentication
```
</details>

## Client configuration

**Browser:** Mozilla/5.0 (X11; Linux x86_64; rv:73.0) Gecko/20100101 Firefox/73.0

**Operating system:** 

## Logs

<details><summary>Web server error log</summary>

```
Insert your web server log here 
```
</details>

<details><summary>Nextcloud log</summary>

```
Insert your Nextcloud log here
```
</details>

<details><summary>Browser log</summary>

Insert your browser log here, this could for example include:

	a) The javascript console log
	b) The network log
	c) ...
</details>

# #
Screenshot_20200309_124109

### PR DESCRIPTION
**Pull request by skjnldsv**
Fix #19859 

Capture d’écran_2020-04-09_09-37-50

Will be better with the overflow count once we migrated tags to vue :woman_shrugging:

### CODE REVIEW

File: core/css/systemtags.scss [hunk 1]
```diff
@@ -72,7 +72,12 @@
 .systemTagsInfoView,
 .systemtags-select2-container {
 	width: 100%;
-	
+
+	.select2-choices	 {
```
Comment (juliushaertl) [inline core/css/systemtags.scss:76]:
```suggestion
	.select2-choices {
```

**Review by juliusknorr — APPROVED** [review]
Spacing issue, otherwise :+1:

**Review by ChristophWurst — APPROVED** [review]

**Review by marcoambrosini — APPROVED** [review]

**Comment by skjnldsv** [comment]
In https://github.com/nextcloud/server/pull/20387

**Pull request merged**